src/v2/view-builder/components/BaseAuthenticatorView.js

```diff
@@ -11,7 +11,7 @@ export const BaseAuthenticatorBeacon = HeaderBeacon.extend({
 
   getBeaconClassName: function() {
     const authenticatorKey = _.result(this, 'authenticatorKey');
-    return getIconClassNameForBeacon(authenticatorKey);
+    return getIconClassNameForBeacon(authenticatorKey, this.options.appState.get('remediations'));
```

Comment:
It looks like this is used to get the remediation name, I would recommend using the framework above and create a function that returns the remediation name (if one exists) and pass that into the method instead of passing the entire remediations array.